### ISSUE TITLE
TypeError: '>' not supported between instances of 'int' and 'str'

### ISSUE THREAD

**Issue by zvowell** [mjordan/islandora_workbench#217]
I'm getting the following error when running workbench --check:

    OK, connection to Drupal at http://localhost:8000 verified.
    OK, configuration file has all required values (did not check for optional values).
    OK, CSV file testBatch_01/barneberg_forTesting_001.csv found.
    OK, all 15 rows in the CSV file have the same number of columns as there are headers (12).
    OK, CSV column headers match Drupal field names.
    OK, required Drupal fields are present in the CSV file.
    OK, ETDF field values in the CSV file validate.
    Traceback (most recent call last):
      File "./workbench", line 1079, in <module>
        check_input(config, args)
      File "/mnt/c/Users/zvowell/islandora8_vmTesting/islandora_workbench/workbench_utils.py", line 807, in check_input
        validate_csv_field_length(config, field_definitions, validate_csv_field_length_csv_data)
      File "/mnt/c/Users/zvowell/islandora8_vmTesting/islandora_workbench/workbench_utils.py", line 1916, in 
    validate_csv_field_length
        if field_name in field_max_lengths and len(
    TypeError: '>' not supported between instances of 'int' and 'str'

Here is the csv I'm using, I tried to simplify it as much as possible to see if that would get past the error: https://gist.github.com/zvowell/5a188b408820cfada36c7f2254d337a7 

And here is my config file:

    task: create
    host: "http://localhost:8000"
    username: admin
    password: islandora
    input_dir: testBatch_01
    input_csv: barneberg_forTesting_001.csv

**Comment by mjordan** [comment]
Thanks for reporting this and providing data and config. I'll investigate, but probably not until the weekend.

**Comment by mjordan** [comment]
@zvowell Strangely, I can't replicate this. The `--check` on a text field's maximum configured length works. But, I've made a change that should fix the problem for you. Can you pull in the most recent changes to the main branch (`git pull origin main`) and run `--check` against the same CSV that made it fail?

**Comment by zvowell** [comment]
@mjordan the change fixed it for me, I'm not getting the error anymore. Thanks! 

As a side note, and I don't know if this is relevant to the change you made for this issue, but when I first ran workbench after updating the repo, I did receive an error that the python module magic was not installed. Once I installed magic, though, the --check process went smoothly.

**Comment by mjordan** [comment]
Very good, thanks for testing and confirming it worked. The absence of the magic library isn't related to the issue you found (I am guessing it was an issue for you specific to the version of Python you are running, since I couldn't replicate it). The problem with the magic library not being installed was just a fluke - I merged PR #215 (which introduced that library) into the main branch before you pulled in updates.

Since your issue has been resolved, I'm going to close this, but feel free to open new issues if you you encounter any.
